docs/design/openshift.md

```diff
@@ -119,9 +119,9 @@ The response from the policy engine will conform to the [Cincinnati Graph API re
 
 ## Update Format ##
 
-### Update Payload ###
+### Update image ###
 
-The Update Payloads served by Cincinnati to the Cluster Version Operator are formatted as a JSON document containing a reference to a container image. This document is versioned so that it may change over time. An example of this document is shown below:
+The Update images served by Cincinnati to the Cluster Version Operator are formatted as a JSON document containing a reference to a container image. This document is versioned so that it may change over time. An example of this document is shown below:
```

Comment:
```suggestion
The Update Images served by Cincinnati to the Cluster Version Operator are formatted as a JSON document containing a reference to a container image. This document is versioned so that it may change over time. An example of this document is shown below:
```